cli/cmd/inject.go

```diff
@@ -80,17 +81,17 @@ sub-folders, or coming from stdin.`,
 				return err
 			}
 
-			configs, err := options.fetchConfigsOrDefault()
+			values, err := options.fetchConfigsOrDefault()
```

Comment:
I think if we pull in https://github.com/linkerd/linkerd2/pull/5038 then we can add `makeProxyFlags` and `makeInjectFlags` from https://github.com/linkerd/linkerd2/pull/5005 to this branch.  This lets us remove `proxyConfigOptions` altogether.